packages/theme-language-server-common/src/TypeSystem.ts

```diff
@@ -809,7 +809,7 @@ function inferArrayTypeLookupType(curr: ArrayType, lookup: LiquidExpression) {
       }
 
       default: {
-        return Untyped;
+        return Unknown;
```

Comment:
`{{ product.images.foo }}` should not have a hover window since we know that `product.images` is an `image[]` and `foo` is not a platform supported feature.